components/motor/motor.go

```diff
@@ -104,8 +104,6 @@ type Motor interface {
 	// revolutions at a given speed in revolutions per minute. Both the RPM and the revolutions
 	// can be assigned negative values to move in a backwards direction. Note: if both are
 	// negative the motor will spin in the forward direction.
-	// If revolutions != 0, this will block until the number of revolutions has been completed or another operation comes in.
-	// Deprecated: If revolutions is 0, this will run the motor at rpm indefinitely.
```

Comment:
Keep the note about blocking, that's important information about this API and part of our API contract.